packages/core/src/browser/dialogs.ts

```diff
@@ -148,15 +148,30 @@ export abstract class AbstractDialog<T> extends BaseWidget {
         this.addKeyListener(document.body, Key.ENTER, e => this.handleEnter(e));
     }
 
+    protected canHandleKeyboardEvent(event: KeyboardEvent): boolean {
+        let result: boolean = false;
+        if (this.id && event.currentTarget) {
+            const dialogOverlay: HTMLElement = event.currentTarget as HTMLElement;
+            if (dialogOverlay.lastElementChild && this.id.localeCompare(dialogOverlay.lastElementChild.id) === 0) {
```

Comment:
How can you be sure that `lastElementChild` belongs to any dialog at all? What if a dialog adds other children to the body?

Reply:
@akosyakov 
I have considered the situation you said, but I want to simplify the problem in the most economical way, so I used a time-stamped ID to determine whether it is the same dialog.